batnode.js

```diff
@@ -417,37 +431,67 @@ class BatNode {
         return true;
       } else {
         this.audit.failed.push(shaId);
-        return false;
       }
     }
-
-    return shaKeys.every(isRedundant);
+    shaKeys.every(isRedundant);*/
+    console.log(auditData, 'audit data')
+    console.log(this.audit.failed, 'this.audit.failed')
```

Comment:
Same with this comment. If it's helpful for debugging you can leave it in, though.